app/models/miq_queue.rb

```diff
@@ -677,14 +677,23 @@ def self.optional_values(options, keys = [:zone])
   private_class_method def self.messaging_options_from_env
     return unless ENV["MESSAGING_HOSTNAME"] && ENV["MESSAGING_PORT"] && ENV["MESSAGING_USERNAME"] && ENV["MESSAGING_PASSWORD"]
 
-    {
+    options = {
       :host     => ENV["MESSAGING_HOSTNAME"],
       :port     => ENV["MESSAGING_PORT"].to_i,
       :username => ENV["MESSAGING_USERNAME"],
       :password => ENV["MESSAGING_PASSWORD"],
       :protocol => ENV.fetch("MESSAGING_PROTOCOL", "Kafka"),
       :encoding => ENV.fetch("MESSAGING_ENCODING", "json")
     }
+
+    if ENV["MESSAGING_KEYSTORE_PASSWORD"].present?
+      options[:ssl] = true
+      options[:ca_file] = "/etc/pki/ca-trust/source/anchors/root.crt"
+      options[:keystore_location] = "/etc/pki/ca-trust/source/anchors/kafka.keystore.jks"
+      options[:keystore_password] = ENV.fetch("MESSAGING_KEYSTORE_PASSWORD", nil)
```

Comment:
Don't think we need the default `, nil)` if we're only entering this conditional `if ENV["MESSAGING_KEYSTORE_PASSWORD"].present?`